common/api/core-bentley.api.md

```diff
@@ -1296,6 +1296,9 @@ export class LRUMap<K, V> extends LRUCache<K, V> {
 // @public
 export type MarkRequired<T, K extends keyof T> = Pick<Required<T>, K> & Omit<T, K>;
 
+// @public
+export type Maybe<T> = T | undefined;
```

Comment:
Nit but do we really need/want this? `x?: X` vs `x: Maybe<X>`. One uses terse, standard TypeScript syntax; the other uses a non-standard type to express the same thing more verbosely. The only place this saves you characters (if that's the goal) is when you want to require someone to specify `undefined` explicitly.